src/start_service.ts

```diff
@@ -38,8 +38,8 @@ export async function startService(opts: {
 
 
   await healthCheck(web3Interface, peerId.toString());
-  fetchProfileAndScheduleUnwraps(web3Interface);
-  setInterval(() => fetchProfileAndScheduleUnwraps(web3Interface), 300000); // refetch every 5mins
+  fetchProfileAndSchedulePublishKeyShares(web3Interface);
```

Comment:
is this scheduler triggering an operation that will publish multiple shares (more than one sarcophagus)? If no maybe call it `fetchProfileAndSchedulePublish` or `fetchProfileAndSchedulePublishKeyShare`

Reply:
@katien -- it is scheduling multiple publishings of key shares --- updated with rename to: `fetchProfileAndSchedulePublish`